spec/system/page_objects/pages/user.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Pages
+    class User < PageObjects::Pages::Base
+      def find(selector)
+        page.find(".user-content-wrapper #{selector}")
+      end
```

Comment:
Since the whole 'user page' is scoped within this wrapper, the page object can scope its finds in this classname